reporting/impl/src/main/resources/reports/templates/techReport-punchCard.ftl

```diff
@@ -189,5 +196,61 @@
     -->
 
     <script src="resources/js/bootstrap.min.js"></script>
+
+    <script>
+        var currentSortColumn = null;
+        var reverse = false;
+
+        function sortTable(column) {
+            if (column == currentSortColumn) {
+                reverse = !reverse;
+            } else {
+                reverse = false;
+                currentSortColumn = column;
+            }
+            var f = reverse ? 1 : -1;
+            var groupedHeaderColumns = $('.headersGroup td').get();
+            groupedHeaderColumns.forEach(function(tdElement) {
+                $(tdElement).removeClass("sectorSorted");
+            });
+            $(groupedHeaderColumns[column]).addClass("sectorSorted");
+
+            var rows = $('.technologiesPunchCard tbody tr').get();
+
+            rows.sort(function(a, b) {
+
+                var A = getVal(a);
+                var B = getVal(b);
+
+                if(A < B) {
+                    return -1*f;
+                }
+                if(A > B) {
+                    return 1*f;
+                }
+                return 0;
+            });
+
+            function getVal(elm) {
+                var v = $(elm).children('td').eq(column).data("count");
+                if($.isNumeric(v)){
+                    v = parseInt(v,10);
+                }
+                return v;
+            }
+
+            $.each(rows, function(index, row) {
+                $('.technologiesPunchCard').children('tbody').append(row);
+            });
+        }
+
+        $().ready(function () {
+            $(".sector").click(function (event) {
```

Comment:
From previous chat we had, this is the selector to change to `.headersGroup .sector `